pkg/util/plugin_schema_helper.go

```diff
@@ -0,0 +1,53 @@
+package util
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/kong/go-kong/kong"
+)
+
+// FIXME
+// decK will release this official API soon, use that and remove this code.
```

Comment:
I know this was just imported from existing code, but can we improve this with an issue link or any other breadcrumbs for the reader?